src/pages/migrating/faq.mdx

```diff
@@ -123,3 +133,193 @@ description:
+
+### What if I want to use a specific theme?
+
+In v11 we introduced a component called `<GlobalTheme/>`. This component makes
+it really easy to change the default theme by using it as a wrapper in your app,
+and passing in a `theme` prop value of `white`, `g10`, `g90`, or `g100`.
+
+### What if I want to do zoned theming or toggle themes?
+
+We recommend using our `<Theme/>` component to include zoned themes. A great
+example of zoned theming using this component lives in our
+[tutorial](https://github.com/carbon-design-system/carbon-tutorial/blob/v11-react-step-5/src/App.js),
+where we themed the header differently than the application theme. If you are
+wanting to toggle themes, for a light and dark mode, we built an
+[example](https://github.com/carbon-design-system/carbon/tree/main/examples/light-dark-mode)
+for you using our `GlobalTheme` component.
+
+### What if I want to use a custom theme?
+
+If you are creating a custom theme or overriding theme tokens, you would do that
+like so:
+
+```scss
+@use '@carbon/react/scss/themes' as *;
+
+//overriding theme tokens and adding a custom token
+@use '@carbon/react/scss/theme' with
+  (
+    $theme: (
+      // add a completely new token
+      custom-token: #bada44,
+      // redefine existing tokens to new values
+      text-error: rebeccapurple,
+      focus: pink,
+    )
+  );
+
+// override a component token
+@use '@carbon/react/scss/components/button/tokens' as button-tokens with
+  ($button-primary: #3f51b5);
+@use '@carbon/react';
+
+:root {
+  @include theme.theme();
+}
+
+// You can access a custom variable via the theme.get() mixin
+.custom-token {
+  background: rgba(theme.get('custom-token'), 0.95);
+}
+```
+
+Check out the full example we built
+[here](https://github.com/carbon-design-system/carbon/tree/main/examples/custom-theme).
+
+### I'm not familiar with sass. How do I learn more?
+
+Hey, we get it. Sass can be a little different if you've never worked with it,
+especially sass modules. We recommend reading thru some of the Sass
+documentation to learn more. A great place to start is
+[Sass Basics](https://sass-lang.com/guide). To learn more about sass modules
+specifically (introduced in v11), we recommend reading the docs for
+[`@use`](https://sass-lang.com/documentation/at-rules/use).
+
+### How do I know what to sass modules to include at the top of my files?
+
+Some of our most used sass assets are included below, along with what you would
+need to import at the top of your file to use it. The `@use` example is
+specifically if you are using our `@carbon/react` package. If you are using our
+`@carbon/styles` the path would be the same, simply replace `react` with
+`styles`. Essentially, all we are doing is including the path where we would
+[find these styles](https://github.com/carbon-design-system/carbon/tree/main/packages/react/scss)
+in our package. We recommend including them using `as *` if you don't want to
+worry about prefixing the sass.
```

Comment:
[Sass docs](https://sass-lang.com/documentation/at-rules/use#:~:text=You%20can%20even%20load%20a%20module%20without%20a%20namespace%20by%20writing%C2%A0%40use%20%22%3Curl%3E%22%20as%20*.%20We%20recommend%20you%20only%20do%20this%20for%20stylesheets%20written%20by%20you%2C%20though%3B%C2%A0otherwise%2C%20they%20may%20introduce%20new%20members%20that%20cause%20name%C2%A0conflicts!) actually recommend against this because they don't control the source of these modules and could introduce conflicts

> You can even load a module without a namespace by writing @use "<url>" as *. We recommend you only do this for stylesheets written by you, though; otherwise, they may introduce new members that cause name conflicts!


```suggestion
in our package. 
```

I honestly think the whole `@use` namespace situation could warrant it's own question in the FAQ. Basically a mirror of the sass docs:
- It includes a namespace by default
- you can rename the namespace, or you can omit it.
- If you omit the namespace, be wary of conflicts.
- Omitting the namespace makes migrating easier so you don't have to add a `namespace.` prefix to every mixin/variable/etc from a module.